src/App.js

```diff
@@ -4,15 +4,57 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>
-          Eric F. Olsen
-        </h1>
-        <h3>
-          Software Engineer
-        </h3>
-      </header>
+      <Header />
+      {/* <Story />
+      <Resume />
+      <Projects />
+      <Testimonials /> */}
+    </div>
+  );
+}
+
+function Header() {
+  return (
+    <header className="App-header">
+      <img src={logo} className="App-logo" alt="logo" />
```

Comment:
Looks like all references to "logo" should now be to your "profile photo"

Reply:
Yeah, I reused logo for simplicity, but I agree it should reflect what it really is. I've implemented this change, now it's either profile-photo (general), profile_photo (on import where hyphen isn't allowed), or "profile photo" (on img alt for accessibility).